backend/app/utils/kafka.py

```
@@ -0,0 +1,23 @@
+                bootstrap_servers=f'{ip_address}:9092',
+                auto_offset_reset='latest',
+                enable_auto_commit=True,
+                group_id=groupid,
+                value_deserializer=lambda x: json.loads(x.decode('utf-8'))
+            )
+            print(f"Kafka consumer for Topic : {{{topic}}} & in Group : {{{groupid}}} initialized.")
+            return consumer
+        except NoBrokersAvailable:
+            print("Kafka broker not available. Retrying in 5 seconds...")
+            time.sleep(5)
+    raise Exception("Kafka broker is not available after multiple attempts.")
```

Ruff (0.15.15)

[warning] Create your own exception

(TRY002)